camp/ocampbell/XBox Trigger Project/XBox Trigger Project/src/main/java/frc/robot/commands/ControlledFalconDefault.java

```diff
@@ -0,0 +1,59 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// File: ControlledFalconDefault.java
+// Intent: Assigns a motor speed value to a falcon.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+package frc.robot.commands;
+
+import frc.robot.control.ManualInputInterfaces;
+import frc.robot.control.SubsystemCollection;
+import frc.robot.subsystems.ControlledFalcon;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ControlledFalconDefault extends CommandBase {
+
+  private SubsystemCollection subsystems;
+
+
+  private ControlledFalcon motor;
+  private double speed = 0.0;
+  private boolean done = false;
+  private double lTrigger = 0.0;
+  private double rTrigger = 0.0;
+
+
+  public ControlledFalconDefault(SubsystemCollection collection, ControlledFalcon motor, double lTrigger, double rTrigger) {
+      this.motor = motor;
+      this.lTrigger = lTrigger;
+      this.rTrigger = rTrigger;
+
+    addRequirements(motor);
+    subsystems = collection;
+}
+
+  @Override
+  public void initialize() {}
+
+
+  @Override
+  public void execute() {
+    lTrigger = ManualInputInterfaces.getInputLeftTrigger();
+    rTrigger = ManualInputInterfaces.getInputRightTrigger();
+
+    speed = (rTrigger+(lTrigger*-1))/2;
```

Comment:
So that means the motor only ever runs at 1/2 speed at best.  If you told John that 'oh you can only drive at 1/2 speed' he'd think that was ok?  Do you think a consumer would want this type of implementation?

When requirements are unclear you need to backup why choosing a specific design/implementation is a good decision.  This area of the code is a reason why I think this mini-project qualifies as 'a second-year type project', namely decisions need to be made on how to synthesize input from two independent controls.  Does the right button win? Does the left button win?  Do you use the relative delta between the two?  If delta approach, why are many movement zeros actually ok?  What do video games do in these situations?  What does the research say?